### PR TITLE
[docs](load) add data type docs

### PR DESCRIPTION
**Pull request by kaijchen**
## Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [ ] 2.0

## Languages

- [x] Chinese
- [x] English

## Docs Checklist

- [ ] Checked by AI
- [ ] Test Cases Built

### CODE REVIEW

File: docs/data-operate/import/data-type/array.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Overview
```
Comment (KassieZ) [inline docs/data-operate/import/data-type/array.md:27]: please start from `## level 2 title`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-type/array.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
+---
+{
+    "title": "ARRAY",
+    "language": "en"
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-type/array.md:4]: zh-CN

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-type/json.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+`JSON` 数据类型，用二进制格式高效存储 JSON 数据，通过 JSON 函数访问其内部字段。
+
+默认支持 1048576 字节（1 MB），可调大到 2147483643 字节（2 GB），可通过 BE 配置`string_type_length_soft_limit_bytes` 调整。
+
+与普通 String 类型存储的 JSON 字符串相比，JSON 类型有两点优势
+
+1. 数据写入时进行 JSON 格式校验
+2. 二进制存储格式更加高效，通过json_extract等函数可以高效访问JSON内部字段，比get_json_xx函数快几倍
+
+:::caution[注意]
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-type/json.md:38]: :::caution 注意

File: sidebars.json [hunk 4]
```diff
@@ -169,6 +169,16 @@
                                         "data-operate/import/import-way/group-commit-manual"
                                     ]
                                 },
+                                {
+                                    "type": "category",
+                                    "label": "Data Type",
```
Comment (KassieZ) [inline sidebars.json:174]: should also edit current.json translate "label": "Data Type" to 数据类型

Comment (KassieZ) [reply sidebars.json:174]: please also double check the difference between reference - sql data type: /sql-manual/sql-data-types/semi-structured/ARRAY

**Comment by wangtianyi2004** [comment]
LGTM

**Pull request merged**